doc/smart-switch/high-availability/smart-switch-ha-dpu-scope-dpu-driven-setup.md

```diff
@@ -137,13 +137,29 @@ The data path and packet format of the DPU-to-DPU probe will be the same as the
 
 In `DPU-Scope-DPU-Driven` setup, the HA state machine will be managed by DPU itself, and `hamgrd` will not drive the HA state machine, but only be used for generating the configurations for NPU and DPU, as well as collecting the telemetry and report in the context of HA whenever is needed.
 
-### 7.1. HA state
+### 7.1. HA states
 
 Since DPU will be driving the HA state machine transition, any HA state change will needs to be reported, otherwise the SDN controller will not be able to know the current HA state of the DPU.
 
-With this setup, all HA states we defined for the ENI-level HA setup can be used to map the undering states, not only `Dead`, `Active`, `Standby` and `Standalone`.
+The DPU driven state machine will be in one of the following operational HA states at any given point
 
-For details on the states and the expected behavior, please refer to the [HA states defined in main HA design doc](./smart-switch-ha-hld.md#71-ha-state-definition-and-behavior).
+ | State |    Description                       |
+ | ----- | ------------------------------------ |
+ | Dead | Initial state not participating in HA |
+ | Connecting | Trying to connect to its HA pair |
+ | Connected | Connection successful, bulk sync in progress |
+ | InitializingToStandalone | Could not connect to pair, waiting for Activate-Role trigger from SDN controller to go to Standalone. Dormant state since BFD disabled hence no traffic. |
+ | InitializingToActive | Bulk sync successful, waiting for Activate-Role trigger from SDN controller to go to Active. Dormant state since BFD disabled hence no traffic. |
+ | InitializingToStandby | Bulk sync successful, waiting for Activate-Role trigger from SDN controller to go to Standby. Dormant state since BFD disabled hence no traffic. |
+ | Standalone | Activate-Role triggered, BFD enabled, forwarding traffic |
+ | Active |  Activate-Role triggered, BFD enabled, forwarding traffic and syncing flows to pair |
+ | Standby |  Activate-Role triggered, BFD enabled, syncing flows from pair |
+ | Destroying | Going down for a planned shutdown |
+ | SwitchingToStandalone | Gracefully transitioning from paired state to standalone |
+
+### 7.2 Activate-Role trigger from SDN controller
+
+When a new DPU is added to a HA set it is required to wait for a trigger from the SDN controller before enabling BFD and starting to participate in data forwarding irrespective of the HA role configured. Before this trigger, the new DPU will wait in one of the InitializingToXXX states where its flow state is completely in sync with its HA pair and it is ready to take up traffic. The SDN controller uses Activate-Role to trigger the DPU HA to enable BFD towards the NPUs and start actively participating in data forwarding.
```

Comment:
no this was a requirement from VNET team

Reply:
i see. let me confirm 1 small thing - if somewhere went wrong in either network or SDN controller, and the current primary is dead so we are failing over to the other side. In this case, we will move to InitializingToStandalone state waiting for activation. And before activation, can the standby create new flows?

Reply:
No ActivateRole is only required once at start-up. If ActivateRole was already triggered for the DPU then it would have been in Active or Standby states. After this if the other DPU dies then this one will immediately move to Standalone.